gradle/publish.gradle

```diff
@@ -1,7 +1,9 @@
 version = '0.5.1'
+group = 'com.novoda.gradle-static-analysis-plugin'
```

Comment:
it seems like this is overlapping with the properties we use to publish to bintray (`groupId` + `artifactId`), what if we define those as local vars here and we compose `group` as convolution (ie: `group = "$groupId.$artifactId"`)?

Reply:
I added them as constants to be reused all over the script